reactor-core/src/main/java/reactor/util/Metrics.java

```diff
@@ -49,4 +52,23 @@ public static final boolean isInstrumentationAvailable() {
 		return isMicrometerAvailable;
 	}
 
+	public static class Configuration {
```

Comment:
Maybe rename `MicrometerConfiguration` ?

Reply:
(the rationale is that this strongly signals classloading of Micrometer. an added benefit is if we ever add support of other metrics frameworks)